main/settings/src/main/scala/sbt/Reference.scala

```diff
@@ -78,7 +78,7 @@ object Reference {
       case LocalRootProject    => "{<this>}<root>"
       case LocalProject(id)    => "{<this>}" + id
       case RootProject(uri)    => "{" + uri + " }<root>"
-      case ProjectRef(uri, id) => "{" + uri + "}" + id
+      case ProjectRef(uri, id) => s"""ProjectRef(uri("$uri"),"$id")"""
```

Comment:
https://github.com/sbt/sbt/pull/3255/commits/41bdf6c8489caafee46f9cd1b34e49bce0a1e7d2 adds `Def.displayRelativeReference`, which displays just the id portion of the `ProjectRef(...)` for the current build. This will be used for the "Updating.. " log. See "Updating root...":

```
hello-world> update
[info] Updating root...
[info] Resolving jline#jline;2.14.3 ...
[info] Done updating.
```